pkg/volume/csi/csi_client_test.go

```diff
@@ -61,6 +69,58 @@ func (c *fakeCsiDriverClient) NodeGetInfo(ctx context.Context) (
 	return resp.GetNodeId(), resp.GetMaxVolumesPerNode(), accessibleTopology, err
 }
 
+func (c *fakeCsiDriverClient) NodeGetVolumeStats(ctx context.Context, volID string, targetPath string) (
+	usageCountMap *volume.Metrics, err error) {
+	var unit csipbv1.VolumeUsage_Unit
+	c.t.Log("calling fake.NodeGetVolumeStats...")
+	req := &csipbv1.NodeGetVolumeStatsRequest{
+		VolumeId:   volID,
+		VolumePath: targetPath,
+	}
+
+	c.t.Logf("c:%+v and nodeclient:%+v", c, c.nodeClient)
+	resp, err := c.nodeClient.NodeGetVolumeStats(ctx, req)
+	c.t.Logf("Response: %v", resp)
```

Comment:
We should not log full object in these tests. In CI logs, they will generate unnecessary noisy output.

Reply:
Removed.